newsfragments/43368.improvement.rst

```diff
@@ -0,0 +1 @@
+``TriggerRule.ALWAYS`` cannot be utilized within a task-generated mapping within mapped task groups
```

Comment:
This note mark as improvment but it feels more like a bug fix? Can we rephrase it to clarify the bug and the fix?

Reply:
We should likely just combine these into a single newfragment too. +1 on bugfix.

Reply:
Combined and renamed into a bug fix, calrified the bug and the fix

Reply:
Had to squeeze it into one line because of the newsfragment pre-commit